apps/ledger-live-mobile/src/components/DeviceAction/InstallSetOfApps/index.tsx

```diff
@@ -38,14 +44,38 @@ const InstallSetOfApps = ({
 }: Props & { remountMe: () => void }) => {
   const [userConfirmed, setUserConfirmed] = useState(false);
   const productName = getDeviceModel(selectedDevice.modelId).productName;
+  const lastSeenDevice: DeviceModelInfo | null | undefined = useSelector(
+    lastSeenDeviceSelector,
+  );
+  const lastDeviceProductName = lastSeenDevice?.modelId
+    ? getDeviceModel(lastSeenDevice?.modelId).productName
+    : lastSeenDevice?.modelId;
+
+  const shouldRestoreApps = useMemo(() => {
```

Comment:
not sure this needs a memoization ? it's a simple check

Reply:
True, I removed the memo